stixcore/processing/L0toL1.py

```diff
@@ -74,6 +81,10 @@ def process_type(files, processor):
     fits_path = Path('/home/shane/fits_test/L0')
     bd = Path('/home/shane/fits_test/')
 
+    # possible set an alternative spice kernel if not the latest should be used
+    SpiceKernelManager.instance = SpiceKernelManager(Path(CONFIG.get("Paths", "spice_kernels")))
```

Comment:
Does this need to be a singleton?

Reply:
hmm right the manager do not really have to be a singleton.

just the Spice object need to be one if we will avoid the pass it around everywhere 

will fix this